pegasus/sites.v3/csedweek.org/views/map.haml

```diff
@@ -8,7 +8,7 @@
         = I18n.t(:csedweek_map_signup)
 
   .row
-    .col-xs-10.col-sm-12
+    .col-sm-12
```

Comment:
Yeah, we removed it earlier on the HOC homepage, so just copying that change here as well. We will guarantee user can scroll the page by capping the height of the map with the latest change. Looks better like this. Looked like like an alignment issue before.